tests/select_for_update/tests.py

```diff
@@ -592,3 +592,8 @@ def test_ordered_select_for_update(self):
         with transaction.atomic():
             qs = Person.objects.filter(id__in=Person.objects.order_by('-id').select_for_update())
             self.assertIn('ORDER BY', str(qs.query))
+
+    @skipUnlessDBFeature('has_select_for_update_of')
```

Comment:
I added `supports_select_for_update_with_limit` because this will crash on Oracle.